environments/shared/recommended.js

```diff
@@ -663,6 +663,12 @@ module.exports = {
 
     // Forbid a module from importing itself
     'import/no-self-import': 'error',
+
+    // Enforce a convention in the order of require() / import statements
+    'import/order': ['error', {
+      // todo: enable moment newslines-between when it will be fixable
```

Comment:
Please make this a `@TODO`. 😄

Also I am not in agreement with having newlines between the groups; I'd rather not have newlines between groups at all or not enforce any particular style. It just looks weird if you have only one import of a particular type in a module and there are more types used, ie.

```js
import fs from 'fs'

import _ from 'lodash'

import Component from './component')
```

So many unnecessary spaces... 🤔

Reply:
good point, removing then